.github/workflows/sdk-pr.yaml

```
@@ -38,4 +38,4 @@ jobs:
           GH_REPO: 'stackitcloud/stackit-sdk-go'
           GH_TOKEN: ${{ secrets.SDK_PR_TOKEN }}
         run: |
-          scripts/sdk-create-pr.sh "generator-bot-${{ github.run_id }}" "Generate services" "Generator: update SDK /services" "Generated from GitHub run ${{ github.run_id }} (${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})"
\ No newline at end of file
+          scripts/sdk-create-pr.sh "generator-bot-${{ github.run_id }}" "Generated from GitHub run ${{ github.run_id }} (${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})"
```

Since we are here, could you update it so that the link is embeded in the text?


```suggestion
          scripts/sdk-create-pr.sh "generator-bot-${{ github.run_id }}" "Generated from GitHub run [${{ github.run_id }}](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})"
```

---

Not sure if this works as we would expect and as it does in the UI box for the PR description